docs/src/confidential-token/deep-dive/overview.md

```diff
@@ -238,7 +238,7 @@ zero-knowledge proofs.
   `lower_bound <= x < upper_bound`.
 
   In the confidential extension, we require that a transfer instruction includes
-  a range proof that certifies the following:
+  a range of proof that certifies the following:
```

Comment:
A "range proof" is a specific type of cryptographic proof. Please revert this change.